tidb-configuration-file.md

```diff
@@ -561,16 +561,16 @@ TiDB 服务状态相关配置。
 
 ## stmt-summary <span class="version-mark">从 v3.0.4 版本开始引入</span>
 
-系统表 `events_statement_summary_by_digest` 的相关配置。
+系统表 [statement summary tables](/statement-summary-tables.md) 的相关配置。
 
 ### max-stmt-count
 
-+ `events_statement_summary_by_digest` 表中保存的 SQL 种类的最大数量。
-+ 默认值：100
++ 系统表 [statement summary tables](/statement-summary-tables.md) 中保存的 SQL 种类的最大数量。
++ 默认值：3000
```

Comment:
@crazycs520 @djshow832 Please help confirm from which v4.0 version this new default value takes effect. Thanks! This is important for user documents.

Reply:
Comment addressed and fixed. PTAL~